### DISCUSSION TITLE
Persisting session cookies

### DISCUSSION THREAD

**Discussion by MinaMatta98** [leptos-rs/leptos#2103] [Q&A]
Hello, 

Was there any way to persist cookies on reload?

**Comment by gbj** [comment]
This is how cookies work by default, isn't it? Could you maybe provide some more context for this question?

**Comment by MinaMatta98** [comment]
I'm using axum login and wanted the cookie to persist on reload. 

I'm surely doing something wrong.

**Reply by gbj** [reply]
I wonder if you're actually creating a cookie? or using some other kind of session storage?

Or do you mean "reload" as in killing the server and restarting it? (In that case I don't know if it's supposed to work and if it does!)

**Reply by MinaMatta98** [reply]
Hello,

I'm not killing the server. I'm just doing an F5 page refresh.

![image](https://github.com/leptos-rs/leptos/assets/81748812/de9ee34a-afc6-4b2c-bd66-b6c2a313ce5e)

**Reply by MinaMatta98** [reply]
Ok, this is definitely not related to leptos, but the login auth framework I was using.

Sorry about this.
